package.json

```diff
@@ -55,7 +55,7 @@
     "uiFramework:documentComponent": "cd packages/kbn-ui-framework && yarn documentComponent",
     "kbn:watch": "node scripts/kibana --dev --logging.json=false",
     "build:types": "tsc --p tsconfig.types.json",
-    "kbn:bootstrap": "yarn build:types"
+    "kbn:bootstrap": "yarn build:types && node scripts/register_git_hook"
```

Comment:
I keep going back and forth, but any thoughts on having called out in the `kbn bootstrap`?

Reply:
I was not able to run it into the preinstall because for now I need to use some `node_modules` on `src/dev/run_register_git_hook.js` only available in the end of the install. I can make some tests in order to try to make this running into the preinstall script. About the bootstrap choice, I thought about running this into the bootstrap in the first place but I just had to run the `node scripts/register_git_hook` inside an exec so I endup to doing it here. However I can also call an exec inside the bootstrap with this if you just prefer